### PR TITLE
python-winsdk: Add package

### PR DESCRIPTION
**Pull request by mimi89999**
I only built it and tested it on `clang64`

### CODE REVIEW

**Comment by Biswa96** [comment]
Is it possible to upstream the patches first? The mingw-w64-python-winsdk/001-Regenerate-pywinrt-bindings.patch file is huge.

**Comment by mimi89999** [comment]
They are upstream, but we would need to ask @dlech to regenerate the bindings in the repo.

**Comment by mimi89999** [comment]
To avoid having such a big patch file, I could also fork the repo and apply the patch there

**Comment by Biswa96** [comment]
> To avoid having such a big patch file, I could also fork the repo and apply the patch there

The package should be built from the actual repository. If the patch is in that upstream repository I can be downloaded from the commit in `source=()` array.

**Comment by mimi89999** [comment]
> > To avoid having such a big patch file, I could also fork the repo and apply the patch there
> 
> The package should be built from the actual repository. If the patch is in that upstream repository I can be downloaded from the commit in `source=()` array.

I know. All the code in `pywinrt/winsdk/` is generated using https://github.com/pywinrt/pywinrt/commit/64d3e0706bf516334db90d7b2cd1e7827c42b0de, so the patches are technically upstream, but the bindings in the https://github.com/pywinrt/python-winsdk would need to be updated. Upstream will probably only update them on the next `pywinrt` release.

**Comment by mimi89999** [comment]
@Biswa96 Yesterday winsdk 1.0.0b9 was released that includes all the required patches for building in Clang MSYS2. I updated the PR and it doesn't include any patch file now.

File: mingw-w64-python-winsdk/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-cc")
+source=("https://pypi.org/packages/source/${_realname::1}/${_realname}/${_realname}-${pkgver}.tar.gz")
+sha256sums=('207d35f9ab445521d4f060b5c1f2181660390cd3db27b32961af7438478fb226')
+
+prepare() {
+  cd "${srcdir}/${_realname}-${pkgver}"
+
+  # Remove due to the lack of required headers
+  rm pywinrt/winsdk/src/py.Windows.Graphics.Capture.Interop.cpp
```
Comment (dlech) [inline mingw-w64-python-winsdk/PKGBUILD:27]: This should no longer be needed.

Comment (Biswa96) [reply mingw-w64-python-winsdk/PKGBUILD:27]: Would you like to explain the reason please?

Comment (dlech) [reply mingw-w64-python-winsdk/PKGBUILD:27]: https://github.com/pywinrt/pywinrt/commit/1c7d7118b3025c14e33fc7d01b8afa30c22b0380

Comment (dlech) [reply mingw-w64-python-winsdk/PKGBUILD:27]: If the header is not found at compile time, the equivalent mingw-compatible interface declaration is provided inline.

Comment (Biswa96) [reply mingw-w64-python-winsdk/PKGBUILD:27]: I can try to add the windows.graphics.capture.interop.h file in mingw-w64. Need some time though.

Comment (mimi89999) [reply mingw-w64-python-winsdk/PKGBUILD:27]: Thanks for especially providing a workaround for MSYS2 and sorry for missing this. I will fix it during the next release. Currently I'm still working on getting it to build on MINGW

**Pull request merged**